scripts/compatibility_matrix.py

```diff
@@ -0,0 +1,252 @@
+                        'joyent/release-{}/usr/src/man/man5/'
+                        'zpool-features.5'.format(ver))
+    return sources
+
+
+def netbsd():
+    url = ('http://cvsweb.netbsd.org/bsdweb.cgi/~checkout~/src/external/'
+           'cddl/osnet/dist/cmd/zpool/zpool-features.7'
+           '?content-type=text/plain&only_with_tag={}')
+    sources = {'main': url.format('MAIN')}
+    with urlopen('https://netbsd.org/releases/') as web:
+        tags = findall(r'href="formal-.+?/NetBSD-(.+?)\.html',
+                       web.read().decode('utf-8', 'ignore'))
+    tags = [(v, 'netbsd-' + v.replace('.', '-') + '-RELEASE') for v in tags]
+    for ver, tag in tags:
+        if int(ver.split('.')[0]) >= 9:
+            sources[ver] = url.format(tag)
+    return sources
+
+openzfs_key = 'OpenZFS<br>(Linux, FreeBSD 13+)'
+sources = {
+        openzfs_key: openzfs(),
+        'FreeBSD pre OpenZFS': freebsd_pre_openzfs(),
+        'OpenZFS on OS X': openzfsonosx(),
+        'OmniOS CE': omniosce(),
+        'Joyent': joyent(),
+        'NetBSD': netbsd(),
+        'Illumos': {
+            'master': 'https://raw.githubusercontent.com/illumos/illumos-gate/'
+                      'master/usr/src/man/man5/zpool-features.5',
+            },
+        # 'OpenZFS on Windows': {
+        #    'master': 'https://raw.githubusercontent.com/openzfsonwindows/ZFSin/master/ZFSin/zfs/man/man5/zpool-features.5',
+        #    },
+        }
+
+features = defaultdict(list)
+readonly = dict()
+
+for name, sub in sources.items():
+    LOG.debug('Work on %s...', name)
+    for ver, url in sub.items():
+        LOG.debug('Get %s...', url)
+        with urlopen(url) as c:
+            if c.getcode() != 200:
+                continue
+            man = c.read().decode('utf-8')
+        for line in man.split('\n'):
+            if line.startswith('.It '):
+                line = line[4:]
+            if line.startswith('GUID'):
+                guid = line.split()[-1]
+                if guid == 'com.intel:allocation_classes':
+                    # This is wrong in the documentation for Illumos and
+                    # FreeBSD.  The actual code in zfeature_common.c uses
+                    # org.zfsonlinux:allocation_classes.
+                    guid = 'org.zfsonlinux:allocation_classes'
+                elif guid == 'org.open-zfs:large_block':
+                    guid += 's'
+                domain, feature = guid.split(':', 1)
+                features[(feature, domain)].append((name, ver))
+            elif line.startswith('READ\\-ONLY COMPATIBLE'):
+                readonly[guid] = (line.split()[-1] == 'yes')
+
+os_sources = sources.copy()
+os_sources.pop(openzfs_key)
+header = list(sorted(os_sources.keys()))
+header.insert(0, openzfs_key)
+header = list(zip(header, (sorted(sources[name],
+              key=lambda x: regex(r'[^0-9]', '', x) or x) for name in header)))
+# TODO(gmelikov): add Sortix?
```

Comment:
No. From what I can tell, Sortix is a toy operating system: https://sortix.org

Reply:
It was in original script as dump "not support anything", and after your comment I've removed even this todo.